modules/rubiconBidAdapter.js

```diff
@@ -269,8 +269,11 @@ export const spec = {
         cpm: ad.cpm || 0,
         dealId: ad.deal,
         ttl: 300, // 5 minutes
-        netRevenue: config.getConfig('rubicon.netRevenue') || false
+        netRevenue: config.getConfig('rubicon.netRevenue') || false,
+        advertiserId: ad.advertiser,
+        networkId: ad.network
```

Comment:
I'd propose that these values get set in a sub-object of the bidResponse. i.e. bidResponse.rp.advertiserId and bidResponse.rp.networkId.

These aren't meant to be values used outside certain RP-specific creative review scenarios.

Reply:
Sounds like a good idea.